sycl/unittests/pi/cuda/test_get_plugin.hpp

```diff
@@ -1,8 +1,9 @@
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
 #pragma once
 
-#include <CL/sycl.hpp>
-#include <CL/sycl/detail/pi.hpp>
-#include <algorithm>
 #include <detail/plugin.hpp>
 
 using namespace cl::sycl;
```

Comment:
Could you remove this using as well, please?

Reply:
Done